delivery_package_fee/__manifest__.py

```diff
@@ -3,12 +3,12 @@
 {
     "name": "Delivery Package Fees",
     "summary": "Add fees on delivered packages on shipping methods",
-    "version": "16.0.1.1.1",
+    "version": "18.0.1.0.0",
     "category": "Delivery",
     "website": "https://github.com/OCA/delivery-carrier",
     "author": "Camptocamp, Odoo Community Association (OCA)",
     "installable": True,
     "license": "AGPL-3",
-    "depends": ["delivery"],
```

Comment:
I am still confused why you did this

Reply:
In this module, we need to inherit `stock.picking`, but the current module `delivery` does not depend on `sale_stock`. Therefore, I have updated `delivery_package_fee` to depend on `stock_delivery`.